### PR TITLE
Add SNMT NvisionF405&F411 unified config file

### PR DESCRIPTION
**Pull request by MrXueyuan**

### CODE REVIEW

**Comment by Asizon** [comment]
This is really wrong, pmease read unified targets manual in our repo.

**Comment by MrXueyuan** [comment]
Sorry，I found some errors in the NVision FPV product configuration file. I will make corrections.

**Comment by haslinghuis** [comment]
@MrXueyuan please do that on the same PR! Read the documentation carefully. Don't use dumps or diffs statements inside the config file.

**Comment by MrXueyuan** [comment]
OK, I corrected some of the code again. Thinks

**Comment by haslinghuis** [comment]
@MrXueyuan please follow instructions here:

https://github.com/betaflight/betaflight/blob/master/docs/TargetMaintenance/CreatingAUnifiedTarget.md

**Comment by MrXueyuan** [comment]
I corrected some of the code again. Please review.

File: configs/default/SNMT-NVISIONF405.config [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+dma pin B00 0
+# pin B00: DMA1 Stream 7 Channel 5
+dma pin B01 0
+# pin B01: DMA1 Stream 2 Channel 5
+
+# mixer
+mixer QUADX
+
+# feature
+feature -RX_PPM
```
Comment (haslinghuis) [inline configs/default/SNMT-NVISIONF405.config:100]: These are not needed

File: configs/default/SNMT-NVISIONF411.config [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+dma pin B07 0
+# pin B07: DMA1 Stream 3 Channel 2
+dma pin A08 2
+# pin A08: DMA2 Stream 3 Channel 6
+
+# mixer
+mixer QUADX
+
+# feature
+feature -RX_PPM
```
Comment (haslinghuis) [inline configs/default/SNMT-NVISIONF411.config:97]: Not needed:
https://github.com/betaflight/unified-targets/pull/549/files#diff-84ccb6dbbadb23d59bfd5c578f2df3462be48afd991f71d3b15c0a36c3c222dbR97-R119

File: configs/default/SNMT-NVISIONF722.config [hunk 3]
```diff
@@ -0,0 +1,174 @@
+# version
+# Betaflight / STM32F405 (S405) 4.2.10 Sep 30 2021 / 11:29:38 (e4b212699) MSP API: 1.43
```
Comment (haslinghuis) [inline configs/default/SNMT-NVISIONF722.config:2]: Wrong MCU. Also the PR does only adding F405 and F411

File: configs/default/SNMT-NVISIONF722.config [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+dma pin B07 0
+# pin B07: DMA1 Stream 3 Channel 2
+dma pin A08 2
+# pin A08: DMA2 Stream 3 Channel 6
+
+# mixer
+mixer QUADX
+
+# feature
+feature -RX_PPM
```
Comment (haslinghuis) [inline configs/default/SNMT-NVISIONF722.config:97]: Remove these lines:
https://github.com/betaflight/unified-targets/pull/549/files#diff-b37b80a009d8fbf654e14bb1bec6e1bd80f58cc3788131695c80ae123f0dadc6R97-R119

File: configs/default/SNMT-NVISIONF405.config [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+feature -DYNAMIC_FILTER
+feature RX_SERIAL
+feature LED_STRIP
+feature OSD
+feature AIRMODE
+feature ANTI_GRAVITY
+feature DYNAMIC_FILTER
+
+# serial
+serial 20 1 115200 57600 0 115200
```
Comment (haslinghuis) [inline configs/default/SNMT-NVISIONF405.config:131]: https://github.com/betaflight/unified-targets/pull/549/files#diff-d33a055792397d139e9eddc57e3e04338b006af367e87c916a0e00291fda675cR131-R137

**Review by haslinghuis — CHANGES_REQUESTED** [review]
There are many more configurations already default.

File: configs/default/SNMT-NVISIONF405.config [hunk 6]
```diff
@@ -0,0 +1,191 @@
+# version
+# Betaflight / STM32F405 (S405) 4.2.10 Sep 30 2021 / 11:29:38 (e4b212699) MSP API: 1.43
+
+board_name NVISIONF405
+manufacturer_id SNMT
```
Comment (DusKing1) [inline configs/default/SNMT-NVISIONF405.config:5]: What's this?

**Review by DusKing1 — CHANGES_REQUESTED** [review]

**Comment by Asizon** [comment]
These config files continues really wrong, please read instructions how to get config from a legacy target @MrXueyuan

**Pull request closed without merging**